apps/cnspec/cmd/scan.go

```diff
@@ -352,7 +352,7 @@ func RunScan(config *scanConfig, scannerOpts ...scan.ScannerOption) (*policy.Rep
 		opts = append(opts, scan.WithUpstream(config.runtime.UpstreamConfig))
 	}
 	if config.runtime.Recording != nil {
```

Comment:
nit: we can remove the entire `if` check, `config.runtime.Recording != nil` can never be false

Reply:
+1, :man_facepalming: overlooked that